webpack.dev.js

```diff
@@ -4,20 +4,24 @@ const merge = require('webpack-merge');
 const path = require('path');
 const common = require('./webpack.common');
 
+const filePath = path.join(__dirname, 'browser/dist');
+
 module.exports = merge(common, {
   mode: 'development',
   resolve: {
     extensions: ['.js'],
   },
   devtool: 'source-map',
   devServer: {
-    contentBase: path.join(__dirname, 'browser/dist'),
+    contentBase: filePath,
     historyApiFallback: true,
     hot: true,
     proxy: {
       '/': 'http://localhost:3000',
     },
     port: 3001,
+    // eslint-disable-next-line no-shadow
```

Comment:
The line below (`writeToDisk: filePath => /index\.html/.test(filePath),`), works similarly to `Array.filter()`. Eslint does not like that I declared `filePath` above then used it again in this function. I am not sure how to get around this rule other than to disable it or write out `path.join(__dirname, 'browser/dist')` in the function (I'm not actually sure if that would work).  Read more about it here. https://webpack.js.org/configuration/dev-server/#devserverwritetodisk-

Alternatively, I can avoid using the function above and have dev-server write all the `.js` bundle files locally as well, but that might get messy and/or slow.  Le me know what you guys think. Thanks!